act-rules-format.md

```diff
@@ -206,6 +218,84 @@ For each accessibility requirement in the mapping, an ACT Rule <em class="rfc211
   </ul></blockquote>
 </aside>
 
+#### Secondary Requirements #### {#secondary-requirements}
+
+A secondary accessibility requirement is a requirement that is correlated with the rule, but for which the rule is not designed to test. The outcome of the rule impacts the result of the accessibility requirement, but the rule is not intended to test the conformance of that requirement. This correlation often results in some of the rule's test cases not satisfying the secondary accessibility requirement.
+
+When the rule is not designed to test the accessibility requirement, or failed outcomes of the rule still require further testing for the accessibility requirement, the rule <em class="rfc2119">may</em> map the accessibility requirement as Secondary. When an ACT rule maps to a Secondary requirement, it <em class="rfc2119">must</em> include an explanation of why that requirement is Secondary in the Background section of the rule.
+
```

Comment:
With the current definition, 2.4.9 is still a Conformance Requirement for any 2.4.4 rule.
I think we need to add something like:
> When requirement B is a stricter version of requirement A, and requirement A is a Conformance requirement; then the rule **must** list requirement B as a Secondary requirement (not as a Conformance requirement).

With "stricter version" being:
> Requirement B is a stricter version of requirement A if every test subject that does not satisfy A also does not satisfy B.